Python/optimizer_bytecodes.c

```diff
@@ -210,136 +210,59 @@ dummy_func(void) {
     }
 
     op(_BINARY_OP_ADD_INT, (left, right -- res)) {
-        if (sym_is_const(ctx, left) && sym_is_const(ctx, right)) {
-            assert(PyLong_CheckExact(sym_get_const(ctx, left)));
-            assert(PyLong_CheckExact(sym_get_const(ctx, right)));
-            PyObject *temp = _PyLong_Add((PyLongObject *)sym_get_const(ctx, left),
-                                         (PyLongObject *)sym_get_const(ctx, right));
-            if (temp == NULL) {
-                goto error;
-            }
-            res = sym_new_const(ctx, temp);
-            Py_DECREF(temp);
-            // TODO gh-115506:
-            // replace opcode with constant propagated one and add tests!
-        }
-        else {
-            res = sym_new_type(ctx, &PyLong_Type);
-        }
+        // We need to tell the cases generator that it's being used by the constant generator.
+        // We should fix this in the cases generator.
+        (void)(left);
+        (void)(right);
```

Comment:
A nitpick so feel free to ignore, but the version without parentheses is more common:
```suggestion
        (void)left;
        (void)right;
```

Reply:
I applied all changes except this one